lib/pp.ml

```diff
@@ -0,0 +1,176 @@
+module List = struct
```

Comment:
We should use `pp` from OPAM--try adding it to the dependencies in `petr4.opam` (the "depends" clause) and `lib/dune` (the `(libraries ...)` list) and deleting this file.